theseus_gui/src/pages/Index.vue

```diff
@@ -40,27 +43,38 @@ const getFeaturedModpacks = async () => {
     `https://api.modrinth.com/v2/search?facets=[["project_type:modpack"]]&limit=10&index=follows&filters=${filter.value}`,
     'featured modpacks'
   )
-  featuredModpacks.value = response.hits
+  if (response) featuredModpacks.value = response.hits
 }
 const getFeaturedMods = async () => {
   const response = await useFetch(
     'https://api.modrinth.com/v2/search?facets=[["project_type:mod"]]&limit=10&index=follows',
     'featured mods'
   )
-  featuredMods.value = response.hits
+  if (response) featuredMods.value = response.hits
 }
 
 await getInstances()
-await Promise.all([getFeaturedModpacks(), getFeaturedMods()])
 
-const unlisten = await profile_listener(async (e) => {
+if (!offline.value) await Promise.all([getFeaturedModpacks(), getFeaturedMods()])
+
```

Comment:
also don't think this is needed- we can just let it fail. eventually we'll be caching this stuff anyways